test/modules/DeploymentTester/Settings.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace DeploymentTester
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Configuration;
+
+    public class Settings
+    {
+        public const string EnvironmentVariablePrefix = "DT_Env";
+
+        static readonly Lazy<Settings> DefaultSettings = new Lazy<Settings>(
+            () =>
+            {
+                IConfiguration configuration = new ConfigurationBuilder()
+                    .SetBasePath(Directory.GetCurrentDirectory())
+                    .AddJsonFile("config/settings.json", optional: true)
+                    .AddEnvironmentVariables()
+                    .Build();
+
+                return new Settings(
+                    configuration.GetValue<string>("IOTEDGE_DEVICEID"),
+                    configuration.GetValue<string>("IOTEDGE_MODULEID"),
+                    configuration.GetValue("DEPLOYMENT_TESTER_MODE", DeploymentTesterMode.Receiver),
+                    configuration.GetValue<string>("IOT_HUB_CONNECTION_STRING"),
```

Comment:
Other modules use ServiceClientConnectionString instead of IOT_HUB_CONNECTION_STRING

Reply:
I think it is odd, as I am not creating a service client.  Should other usage use IOT_HUB_CONNECTION_STRING?  I am thinking to change other usages in another PR, what do you think?

Reply:
I found that in new test framework, IOT_HUB_CONNECTION_STRING is used.  https://github.com/Azure/iotedge/blob/739eb5852400e75b2327df293e2ae856108e0d0e/test/Microsoft.Azure.Devices.Edge.Test/helpers/Context.cs#L82
should we just use it as well?

Reply:
yes, let's use it